packages/typescript/test/ts_project/json/BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "generated_file_test")
+load("//packages/typescript:index.bzl", "ts_project")
+
+SRCS = glob(["**/*.ts*"]) + [
```

Comment:
aside: interesting pattern rather than `["**/*.ts", "**/*.tsx"]` - I just wish the glob pattern allowed `["**/*.ts?"]` so this wouldn't accidentally overmatch. But not much starts with "ts" so I'm considering adopting this in react projects...